drivers/display/display_sdl.c

```diff
@@ -91,11 +129,48 @@ static int sdl_display_init(const struct device *dev)
 
 	if (rc != 0) {
 		LOG_ERR("Failed to create SDL display");
-		return -EIO;
+		return;
```

Comment:
Honest question: Do you expect anything sensible will happen from here on? or should this just `nsi_print_error_and_exit()` here?

Reply:
Was not sure if the `nsi_` function are to be used within the driver, will change.

Reply:
Done.